pixi/src/ui/PageExperience.js

```diff
@@ -145,7 +145,7 @@ export default class PageExperience {
     if (error) {
       console.error('Could not perform AMP Linter check', error);
     }
-    this.reportViews.httpsCheck.render(data.result);
+    this.reportViews.httpsCheck.render(data.usesHttps);
```

Comment:
Hey @caroqliu, wouldn't it be more consistent to stick with `data.result` here? I think I've missed adapting the changes [here](https://github.com/ampproject/amp.dev/pull/4355/files#r476282034)

Reply:
Sg, I'll remove this change entirely as t his is fixed in #4372